features/collection/loaders/index.ts

```diff
@@ -205,21 +203,15 @@ interface PostImage {
    'rp4wp-thumbnail-post': string;
 }
 
-async function loadRelatedNews(tags: string[]): Promise<News[]> {
-   const base64Credentials = convertToBase64(
-      `${WP_BASIC_AUTH_USER}:${WP_BASIC_AUTH_PASSWORD}`
+async function loadRelatedNews(tags: string[]): Promise<Post[]> {
```

Comment:
Minor: We could rename this method to `loadRelatedPosts` since we seem to be using that terminology now.